foreman/data_refinery_foreman/foreman/management/commands/create_missing_processor_jobs.py

```diff
@@ -0,0 +1,193 @@
+"""This command will find successful downloader jobs who never
+spawned a processor job and will create processor jobs for them. This
+is most likely to be a temporary patch until we can figure out why
+downloader jobs are failing to create processor jobs.
+"""
+
+from typing import List
+
+from django.core.management.base import BaseCommand
+from data_refinery_common.job_lookup import (
+    ProcessorPipeline,
+    determine_processor_pipeline,
+    determine_ram_amount,
+    is_file_rnaseq,
+)
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_common.message_queue import send_job
+from data_refinery_common.models import (
+    DownloaderJob,
+    ProcessorJob,
+    OriginalFile,
+    DownloaderJobOriginalFileAssociation,
+    ProcessorJobOriginalFileAssociation
+)
+
+
+logger = get_and_configure_logger(__name__)
+
+BLACKLISTED_EXTENSIONS = ["xml", "chp", "exp"]
```

Comment:
There are more than this

Reply:
I copied this from here https://github.com/AlexsLemonade/refinebio/blob/dev/workers/data_refinery_workers/downloaders/utils.py#L32 (since I didn't wanna refactor stuff out of the workers sub-project to the common just for a temporary change).

What are these additional blacklisted extensions and where is it being enforced? Should it be added to https://github.com/AlexsLemonade/refinebio/blob/dev/workers/data_refinery_workers/downloaders/utils.py#L32?